src/Controller/ApiController.php

```diff
@@ -160,6 +160,25 @@ public function debugAction(Request $request)
         return $this->createJobFromParams($params);
     }
 
+    /**
+     *  Debug - create a snapshot of data folder before and after every container, do not perform output mapping.
+     *
+     * @param Request $request
+     * @return \Symfony\Component\HttpFoundation\Response
+     * @throws \Exception
+     */
+    public function debugBranchAction(Request $request)
+    {
+        // Get params from request
+        $params = $this->getPostJson($request);
+        $component = $request->get("component");
```

Comment:
tu by som dal radsej single quotes okolo `component`

Reply:
jasne, to je nejakej copy paste